### ISSUE TITLE
using mbtiles from another drive/partition fails silently on windows

### ISSUE THREAD

**Issue by indus** [klokantech/tileserver-gl#228]
> - Clear description of the problem: What steps will lead to reproducing the error on our computer? What is exactly wrong?

liveserver-gl-light fails silently when using an mbtiles file from another drive/partion then the one you have started it in.

On a windows PC: Place your mbtiles file on your secondary drive/partition (lets say drive letter D) then open a CMD on you main drive/partition (lets say drive letter C), start liveserver-gl-light from C opening the file on D and try to open the map with your browser. The map stays empty.

> - Version of the TileServer GL software you have used

2.2.0

> - Version and name of the operating system you use or other details of your setup

Windows 7

> - Information about your used config / styles / vector tiles

This fails with and without config in the same way.

**Comment by indus** [comment]
any updates?

**Comment by petrsloup** [comment]
This looks like the file is not a valid mbtiles. This usually happens if the file was not downloaded correctly and is incomplete.

Why do you think it's a Windows-specific issue? Have you tried the same file on the same partition? Is there any difference? Have you tried it on other platform?

**Comment by indus** [comment]
When I use the very same file from the same partition it works without any problem. It just doesn't work when the tileserver-gl is startet from another partition/ with another context.

I'm not saying this is windows specific - but I'm facing this bug on windows.

**Comment by chattes** [comment]
Any updates on this issue? I am trying to load .mbtiles from a mounted partition(NAS) and get the same error? It runs fine when on the same partition drive. This is in Ubuntu?

**Comment by petrsloup** [comment]
I don't think this is issue with tileserver-gl, but maybe an issue with correct file permissions?
(I'm not able to test this now since I don't have a Windows machine set up.)

**Comment by chattes** [comment]
Thank you but I tried this on an Ubuntu Server

**Comment by MrDrooky** [comment]
I ran into the same issue when testing the server on a macosx with docker and accessing files from a FAT32 usb.  It was fixed when I reformatted to OSX extended.  I didn't explicitly change any permissions.
